packages/react/lib/Editable/Tab.tsx

```diff
@@ -83,7 +84,11 @@ const Tab = ({
         .concat(componentOverride?.fields?.filter(f =>
           !component.settings?.fields?.find(s =>
             s.type !== 'tab' &&
-            (s as ComponentSettingsFieldObject).key === f.key
+            ((s as ComponentSettingsFieldObject).key === f.key ||
+            areArraysEqual(
```

Comment:
key can also be a string on one side only (ex: s.key is a string, f.key is an array), `areArraysEqual` seems to exclude that from the equation, I think the real comparison should be `[].concat(s.key).some(k => [].concat(f.key).includes(k))` or something less vomit inducing